### PR TITLE
Fix RefPack errors

### PR DESCRIPTION
**Pull request by feliwir**

### CODE REVIEW

File: src/game/common/compression/refpack.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ static uint32_t RefPack_Matchlen(const uint8_t *s, const uint8_t *d, uint32_t ma
 /**
  * Compresses data using refpack LZ method
  */
-static int RefPack_Encode(const void *src, int src_len, void *dst, int dst_len, bool quick)
+static int RefPack_Encode(const void *src, int src_len, void *dst, int dst_len, int *opts)
```
Comment (tomsons26) [inline src/game/common/compression/refpack.cpp]: opts isn't passed on further, this is quick in this function
and quick in RefPack_Uncompress is passed as false

**Comment by xezon** [comment]
I suggest test Ref Pack compress and uncompress against original compressed .map file. Uncompress it first, then compress it again, then binary compare original .map with new .map file. If files are a match, then Ref Pack is success.

**Comment by feliwir** [comment]
> I suggest test Ref Pack compress and uncompress against original compressed .map file. Uncompress it first, then compress it again, then binary compare original .map with new .map file. If files are a match, then Ref Pack is success.

compression tests are added in my zlib PR (#791)

**Comment by xezon** [comment]
> compression tests are added in my zlib PR (#791)

To clarify, the request here is to check Ref Pack correctness against original game map data. Testing Ref Pack against artificial test data will only check correctness against itself, but not against third party compressed data.

**Review by tomsons26 — APPROVED** [review]

**Comment by xezon** [comment]
Merge this?

**Pull request merged**